inst/WORDLIST

```diff
@@ -14,8 +15,8 @@ customizations
 de
 df
 droppage
+formatter
 funder
-getter
```

Comment:
I am wondering if we can avoid using these in the docs as it is a term related to OOP and not really of concern for the users. I would devise it to code comments